notebooks/COS/ViewData/ViewData.ipynb

```diff
@@ -26,9 +26,9 @@
     "    \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #61.    ax1.set_ylabel(r'Estimated $\dfrac{SIGNAL}{NOISE}$ for a resolution element',
</pre><p>Similar to above, to fix the error, you can either use a double backslash to represent a single backslash character, or use a raw string.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>